engines/wintermute/utils/path_util.cpp

```diff
@@ -73,12 +73,7 @@ Common::String PathUtil::getDirectoryName(const Common::String &path) {
 //////////////////////////////////////////////////////////////////////////
 Common::String PathUtil::getFileName(const Common::String &path) {
 	Common::String newPath = unifySeparators(path);
-	Common::String lastPart = Common::lastPathComponent(newPath, '/');
-	if (lastPart[lastPart.size() - 1 ] != '/') {
-		return lastPart;
-	} else {
-		return path;
-	}
+	return Common::lastPathComponent(newPath, '/');
```

Comment:
Now it does. By the way, is this intended behaviour?
I don't think accessing stuff that ends with a / inside .DCPs is defined behaviour, or is it?